pytorch/jobs/docker/1.9/py3/Dockerfile.gpu

```diff
@@ -136,28 +106,28 @@ RUN ${PIP} install --no-cache --upgrade \
         scikit-learn==0.20.3 \
         six==1.15.0 \
         scipy==1.7.3 \
-        torch==1.6.0 \
         typing_extensions==3.7.4.3
 
 RUN ${PIP} install --no-cache --upgrade sagemaker-training
 
-# Some packages can not be installed using pip (because they are not
-# python managed packages) - install them using apt get
-RUN apt-get update && apt-get install -y \
```

Comment:
It's not clear why we no longer need openbabel and psi4

Reply:
These will still be in the base container